build.gradle

```diff
@@ -1,13 +1,41 @@
+import java.text.SimpleDateFormat
+
 plugins {
-    id 'eu.xenit.docker' version '5.1.0' apply false
-    id 'eu.xenit.docker-compose' version '5.1.0' apply false
+    id 'eu.xenit.docker' version '5.1.1' apply false
+    id 'eu.xenit.docker-compose' version '5.1.1' apply false
 }
 
 repositories {
     mavenCentral()
     jcenter()
 }
 
+String getBranchName(Project project) {
+    return project.hasProperty('branchName') ? project.branchName : (System.getenv('BRANCH_NAME') ?: 'local')
+}
+
+String getTagQualifier(String branchName) {
+    if (branchName.endsWith('release')) {
+        return ''
+    }
+
+    if (branchName.endsWith('master')) {
+        return '-SNAPSHOT'
+    }
+
+    return '-' + branchName.split('/').last() + '-SNAPSHOT'
+}
+
+String getBuildTime(String branchName) {
+    def dateFormatPattern = "-yyyyMMdd"
+    if (!branchName.endsWith('release')) {
+        dateFormatPattern += ".HHmm"
```

Comment:
Question for @tgeens / @thijslemmens: Is this the format we are going to use for all general docker images?
In that case, I should roll it into the plugins, since [`autotag.legacyTags()`](https://github.com/xenit-eu/alfresco-docker-gradle-plugin#autotaglegacytagsliststring) is looking for some friends to make everyone's life easier (before this gets copy-pasted to `n^10` projects)

Reply:
Okay, so I guess this is not the format we are going to use.